spec/feature/confirm_email_prompt_spec.rb

```diff
@@ -0,0 +1,111 @@
+require "pry"
```

Comment:
Not needed

Reply:
urgh that was stupid, sorry left over from some debuggin I was just doing.
I might add the `pry-rails` gem under development to help with this :thinking:

Rubocop used to complain when I did silly things like this too...